.travis.yml

```diff
@@ -0,0 +1,86 @@
+os: linux
```

Comment:
actually why not take this opportunity to use GA for deployment? if not we'll have to migrate again in future

Reply:
this is just copy pasted from the old code; still waiting on @tshuli for the GA portion, didn't want to supersede his work